.github/workflows/continuous-integration.yml

```
@@ -113,7 +113,7 @@ jobs:
           fi
       - name: "JS Lint"
         run: |
-          if [[ -f "node_modules/.bin/eslint" && -f ".eslintrc.js" ]]; then
+          if [[ -f "node_modules/.bin/eslint" && [-f "eslint.config.mjs" || -f ".eslintrc.js"] ]]; then
```

It may be easier to read.
```suggestion
          if [[ -f "node_modules/.bin/eslint" && ( -f "eslint.config.mjs" || -f ".eslintrc.js" ) ]]; then
```